source/read/change-streams.txt

```diff
@@ -84,31 +84,23 @@ as it occurs. The printed change event resembles the
 following:
 
 .. code-block:: json
-   :emphasize-lines: 14-20
 
    {
-     "cursor": {
-       "nextBatch": [
-         {
-           "_id": { ... },
-           "operationType": "update",
-           "clusterTime": { ... },
-           ...
-           "ns": {
-             "db": "sample_restaurants",
-             "coll": "restaurants"
-           },
-           ...
-           "updateDescription": {
-             "updatedFields": {
-               "cuisine": "Irish"
-             },
-             "removedFields": [],
-             "truncatedArrays": []
-           }
-         }
-       ],
-       ...
+     "_id": { ... },
+     "operationType": "update",
+     "clusterTime": { ... },
+     ...
+     "ns": {
+       "db": "sample_restaurants",
+       "coll": "restaurants"
+     },
+     ...
```

Comment:
What are these ellipses referring to, and is it necessary to have two sets of them?

Reply:
In this case, I was simply cutting out bits from the result I received when I was testing this myself and replacing them with ellipses.

I can update it to be just one occurrence!